### PR TITLE
VACMS-14961: fix for remaining deprecations in custom code

### LINKED ISSUE #14961: Fix remaining deprecations in custom code
## User Story or Problem Statement
Fix remaining deprecations identified by upgrade status module in preparation for D10. 

## Acceptance Criteria
- [ ] Upgrade Status show all custom modules as D10 ready.

## Team
Please check the team(s) that will do this work.

- [x] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] User support
- [ ] Accelerated Publishing

### PR DESCRIPTION
**Pull request by edmund-dunn**
## Description

Closes #14961.

Still shows the core/ckeditor library as deprecated  in Upgrade status report for vagovclaro. That is expected because we are launching ckeditor5 with d10. 

## Testing done
Tested preview locally. Works as expected. 

## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

1. Check Upgrade status report
   - [x] Validate that only 1 item remains under the Fix manually section. 
2. Then find a published piece of content
   - [x] Validate that the preview works.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/modules/custom/va_gov_preview/src/Encoder/StaticEncoder.php
Comment (ndouglas) [inline docroot/modules/custom/va_gov_preview/src/Encoder/StaticEncoder.php:1]: Does this pose a risk to preview server functionality?

**Review by ndouglas — COMMENTED** [review]
This looks great!

**Comment by ndouglas** [comment]
Does this have any specific QA steps or just click around and see? 👀

File: docroot/themes/custom/vagovclaro/assets/js/inline_guidance.js [hunk 1]
```diff
@@ -4,28 +4,27 @@
...
+          setTimeout(() => {
             $("#inline-guidance-trigger").focus();
           }, 500);
         }
       });
     }
   };
-})(jQuery, window.Drupal);
\ No newline at end of file
+})(jQuery, Drupal, window.once);
```
[the remark this answers is not in the document]
Comment (edmund-dunn) [reply docroot/themes/custom/vagovclaro/assets/js/inline_guidance.js:30]: good catch, it should be `window.Drupal`

**Comment by edmund-dunn** [comment]
> Does this have any specific QA steps or just click around and see? 👀

There are steps, see above.

**Comment by ndouglas** [comment]
well, I'm an idiot

**Review by ndouglas — APPROVED** [review]
Looks like test-prod can't be used to test preview-server-facing changes (😩), assuming test-staging can't either. staging itself can be used, though, so let's see if we can confirm this works on staging 🙂

**Pull request merged**